tests/manage/mcg/test_mcg_log_level.py

```diff
@@ -0,0 +1,115 @@
+import pytest
+
+from logging import getLogger
+
+from ocs_ci.framework import config
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.pod import (
+    get_noobaa_core_pod,
+    get_noobaa_endpoint_pod,
+    get_noobaa_operator_pod,
+    get_pod_logs,
+)
+from ocs_ci.framework.testlib import tier2, bugzilla
+from ocs_ci.ocs.constants import (
+    CONFIGMAP,
+    NOOBAA_CONFIGMAP,
+    POD,
+    NOOBAA_APP_LABEL,
+    STATUS_RUNNING,
+    OPENSHIFT_STORAGE_NAMESPACE,
+)
+
+log = getLogger(__name__)
+NAMESPACE = config.ENV_DATA.get("cluster_namespace", OPENSHIFT_STORAGE_NAMESPACE)
+LOG_LEVEL_DEFAULT = "default_level"
+LOG_LEVEL_WARN = "warn"
+LOG_LEVEL_ALL = "all"
+
+
+def get_noobaa_cfg_log_level(cfgmap: OCP) -> str:
+    """
+    Gets noobaa current log level from its configmap
+
+    Args:
+        cfgmap (OCP): OCP object of kind "configmap"
+
+    Returns:
+        str: value of NOOBAA_LOG_LEVEL
+
+    """
+    log_level = cfgmap.get(NOOBAA_CONFIGMAP).get("data").get("NOOBAA_LOG_LEVEL")
+    log.info(f"Noobaa current log level from configmap: {log_level}")
+    return log_level
+
+
+def set_noobaa_cfg_log_level(cfgmap: OCP, log_level: str) -> None:
```

Comment:
use `change_the_noobaa_log_level` fixture instead

Reply:
this fixture does not enable any flexibility, so i would like to keep these get/set functions. i will move it out of /test/ folder